india_compliance/tests/test_records.json

```diff
@@ -344,6 +344,25 @@
                     "link_name": "_Test Registered Composition Supplier"
                 }
             ]
+        },
+        {
+            "name": "_Test Indian Registered Company-Shipping",
+            "address_type": "Shipping",
+            "address_line1": "Test Address - 5",
+            "city": "Test City",
+            "state": "Gujarat",
+            "pincode": "380015",
+            "country": "India",
+            "gstin": "24AAQCA8719H1ZC",
+            "gst_category": "Registered Regular",
+            "is_shipping_address": 1,
```

Comment:
This can also be dropped if not required. Could conflict.

Reply:
It's required to check different billing and shipping address